contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.3.0--2.4.0.sql

```diff
@@ -162,6 +162,9330 @@ WHERE has_schema_privilege(s.schema_id, 'USAGE')
 AND c.contype = 'c' and c.conrelid != 0;
 GRANT SELECT ON sys.check_constraints TO PUBLIC;
 
+-- make sys functions stable
+CREATE OR REPLACE FUNCTION sys.schema_id()
```

Comment:
Use ALTER FUNCTION statement to change VOLATILE function to STABLE. For example,
```
ALTER FUNCTION sys.schema_id() STABLE;
```

Redefining so many functions is error-prone and the git commit context of individual functions will be lost.

Reply:
Have done the above changes